docker/hugo/snippets/common

```diff
@@ -1,8 +1,5 @@
 ARG HUGO_EXTENDED=
 
-# gcc is required by extended Hugo.
-RUN if [[ -n "$HUGO_EXTENDED" ]]; then apk add --no-cache gcc; fi
```

Comment:
Based on the error, it looks like you need to add `libstdc++`, which contains the C++ runtime:

```
apk add --no-cache libstdc++
```

After installing `libstdc++` on alpine, the missing shared libraries become available:

```
/ # ls /usr/lib
engines-3            libgcc_s.so.1        libstdc++.so.6       modules-load.d
libcrypto.so.3       libssl.so.3          libstdc++.so.6.0.32  ossl-modules
```

Reply:
You're right, I found the reference  https://github.com/gohugoio/hugo/blob/2da4ec5738e9b9060aab37247312244e1b8a318d/Dockerfile#L33-L36.

Let me know if this PR is OK, will merge and rebuild images for latest version